openpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -111,11 +112,16 @@ def process(self, instance):
             "Files will be rendered to folder: {}".format(output_dir)
         )
 
+        result = None
         if instance.data["family"] == "review":
             result = self.render_review(
                 output_dir, mark_in, mark_out, scene_bg_color
             )
-        else:
+        elif self.extract_json and instance.data['family'] == 'renderLayer':
+            self.add_render_layer_to_repre(instance)
+            return
```

Comment:
So whenever extract json is enabled it doesn't render but uses existing frames? Some logic here seems off?

It should just be able to render AND generate the JSON, right?